docker/Dockerfile.spdk

```diff
@@ -0,0 +1,22 @@
+FROM fedora:33 AS base
```

Comment:
What is the reason for having a separate spdk container?

Reply:
The issue I was facing is every time we make a gateway code change and build the container, spdk was getting build.
Mounting a pre-built spdk directory into the container forces us to develop on linux.
A separate spdk container helps us avoid both issues.

Reply:
make sense